### PR TITLE
Added Summit location and fixed YAML errors for OSD meetup recaps

### PR DESCRIPTION
**Pull request by AnXh3L0**
As the title says, I added the location for the 2018 Summit and fixed some YAML rendering errors for 2 OSD meetup recaps that don't show up in the homepage.

### CODE REVIEW

**Comment by ei8fdb** [comment]
Small suggestion: maybe have the OSM link open in a new window? 

Everything else is 👍.

File: summit/2018.html [hunk 1]
```diff
@@ -33,8 +33,8 @@ <h2 class="location">Tirana, Albania
         <img class="marker" src="/images/summit/map-marker.svg" width="55px" height="74px"
             alt="Location"/>
         <h2 class="location">Location</h2>
-        <p class="data">Venue still to be confirmed<br />Tirana, Albania</p>
-        <!--<img class="map" src="/images/summit/map.jpg" width="540px" height="313px" alt="Map"/>  -->
+        <p class="data"><strong>OFIÇINA</strong><br />Rruga Asim Vokshi<br />Tirana, Albania</p>
+        <a href="https://osm.org/go/xexSY_f5h--?m=" target="_blank" rel="noopener"><img class="map" src="/images/summit/map.jpg" width="600px" height="347px" alt="Map"/></a>
```
Comment (ei8fdb) [inline summit/2018.html]: Suggest making: <a href="https://osm.org/go/xexSY_f5h--?m=" target="_blank"

into 

        <a href="https://osm.org/go/xexSY_f5h--?m=" target="_new"

...

Comment (elioqoshi) [reply summit/2018.html]: Good thinking!

Comment (jancborchardt) [reply summit/2018.html]: @ei8fdb @elioqoshi thought I missed something – but "_new" does not exist ;) https://stackoverflow.com/a/4964223

**Pull request merged**